### PR TITLE
Use GrubConf combiner in SELinux

### PR DESCRIPTION
**Pull request by xiangce**
- fix: #2138
Signed-off-by: Xiangce Liu <xiangceliu@redhat.com>

### CODE REVIEW

File: insights/combiners/selinux.py [hunk 1]
```diff
@@ -39,17 +39,16 @@
 BOOT_NOT_ENFORCING = 'selinux_conf_not_enforcing'
 
 
-@combiner(SEStatus, SelinuxConfig,
-          optional=[Grub1Config, Grub1EFIConfig, Grub2Config, Grub2EFIConfig])
+@combiner(SEStatus, SelinuxConfig, GrubConf)
```
Comment (bfahr) [inline insights/combiners/selinux.py]: @xiangce the original logic had grub parsers as optionals, you've made the combiner mandatory here.  The logic still checks for the existence of a grub combiner so I think it should still be optional.

Comment (xiangce) [reply insights/combiners/selinux.py]: Done. Thanks @bfahr

**Comment by bfahr** [comment]
@jsvob @skontar can you take a look at this PR that was merged.  It appears to have broken prodsec rules.

**Comment by bfahr** [comment]
@jsvob @skontar nevermind, I believe @kgrant-rh  fixed the problem.

**Pull request merged**